roles/rabbitmq/handlers/main.yml

```diff
@@ -1,2 +1,8 @@
-  - name: restart rabbitmq
-    action: service name=rabbitmq-server state=restarted enabled=yes
+---
+- name: restart rabbitmq
+  action: service name=rabbitmq-server state=restarted enabled=yes
+
+- name: login as rabbitmq to update limits
+  command: sudo -i -u rabbitmq
+  register: rabbitmq_login
+  failed_when: rabbitmq_login.rc != 1
```

Comment:
Avoids a reboot to pick up `/etc/security/limits.conf` and `limits.d` changes.  Changes are picked up at login; sudo processes a login with `-i`; shell is `/bin/false` which returns a 1